http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/models.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import java.time.Instant
+import java.util.UUID
+
+import cats.effect.{IO, Timer}
+import org.broadinstitute.dsde.workbench.leonardo.http.nowInstant
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+
+final case class AppContext(traceId: TraceId, now: Instant)
```

Comment:
So `AppContext` can now be used everywhere instead of needing to convert across different types of `ApplicativeAsk`? I like this.

Reply:
yep..that's the idea, I think it should be used in most routes api calls going forward.